src/node_metadata.cc

```diff
@@ -10,7 +10,9 @@
 
 #if HAVE_OPENSSL
 #include <openssl/opensslv.h>
-#include <ngtcp2/ngtcp2.h>
+#if defined(NODE_EXPERIMENTAL_QUIC)
+#include <ngtcp2/version.h>
```

Comment:
?
```suggestion
#include <ngtcp2/ngtcp2.h>
```

Reply:
either works, version makes it more explicit what we're pulling in here